pkg/operator/status.go

```diff
@@ -289,9 +299,172 @@ func (optr *Operator) syncUpgradeableStatus() error {
 		coStatus.Message = "One or more machine config pools are updating, please see `oc get mcp` for further details"
 	}
 
+	// don't overwrite status if updating or degraded
+	if !updating && !degraded {
+		skewStatus, status, err := optr.isKubeletSkewSupported(pools)
+		if err != nil {
+			coStatus.Reason = status.Reason
+			coStatus.Message = status.Message
+			return optr.updateStatus(co, coStatus)
+		}
+		glog.Infof("kubelet skew status: %v, status reason: %v, status message: %v", skewStatus, status.Reason, status.Message)
```

Comment:
I think we should not log message when there is no version skew, otherwise operator log will be flooded with the message while user doesn't have to take any action. Related to https://github.com/openshift/machine-config-operator/pull/2658/files#r675479794 . We may have to do same for `skewSupported` as well but that can be done later on as well.